### ISSUE TITLE
muPSNR too low

### ISSUE THREAD

**Issue by wxy1234567** [liuzhen03/ADNet#14]
I use ID less than 1400 as the training set and the rest as the verification set. The PSNR on the validation set can reach 40 +, but the muPSNR is only 22. In addition, I tested that the muPSNR on 100 training sets can reach 37. The training of the model is not over, but the growth trend of muPSNR on the validation set is not obvious, which seems to be a little over fitting.
